Makefile

```
@@ -10,17 +10,23 @@ help:  ## display this help
 	@cat $(MAKEFILE_LIST) | grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' | \
 		sort -k1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-clean:  ## clean vagrant boxes
+clean:  ## cleanup
+	rm -f requirements.yml
 	rm -f doc/adfinis-sygroup.*.rst
 	$(shell \
 		cd doc; \
 		make clean; \
 		cd ..; \
 	)
-	vagrant destroy -f
 
-test:  ## run test environment
-	vagrant up --provision
+requirements: ## create requirements.yml
+	echo "---" > requirements.yml
+	for role in $$(ansible-galaxy search --author adfinis-sygroup | grep adfinis | cut -d ' ' -f2); do \
```

You could lose a `|` by using awk:
```suggestion
	for role in $$(ansible-galaxy search --author adfinis-sygroup | awk '/adfinis/ {print $2}'); do \
```

---

Fixed and rebased. Note that it has to be the first variable though. ;)

---

yeah, I don't really `cut` by heart like I `awk`